bundles/org.openhab.core/src/main/java/org/openhab/core/library/items/CallItem.java

```diff
@@ -53,9 +54,18 @@ public List<Class<? extends Command>> getAcceptedCommandTypes() {
     @Override
     public void setState(State state) {
         if (isAcceptedState(ACCEPTED_DATA_TYPES, state)) {
-            super.setState(state);
+            applyState(state);
         } else {
             logSetTypeError(state);
         }
     }
+
+    @Override
+    public void setTimeSeries(TimeSeries timeSeries) {
+        if (timeSeries.getStates().allMatch(s -> s.state() instanceof StringListType)) {
```

Comment:
A comment for all the type checks: Should we also allow `UnDefType.NULL` as a valid state, so that time series could also define future periods without a valid state?

Reply:
Shouldn't the check be better done with `isAcceptedState()` instead of `instanceof`?